iceoryx_hoofs/include/iceoryx_hoofs/internal/concurrent/active_object.hpp

```diff
@@ -33,7 +33,7 @@ class ActiveObject
     ActiveObject() noexcept;
     virtual ~ActiveObject() noexcept;
     void addTask(const std::function<void()>& f) noexcept;
-    void mainLoop() noexcept;
+    void mainLoop();
```

Comment:
+1

One more addition: If it's because of `std::thread`. Long-term we want to move away and plan to use a [custom implementation](https://discourse.ros.org/t/safety-working-group-june-presentation-writing-safer-code/20362/4?u=moss) (find more info in the slides).